js/app.js

```diff
@@ -59,7 +59,7 @@
 						var results = [];
 
 						//Add custom entry to create a new empty room
-						results.push({ id: "create-public-room", displayName: t('spreed', 'Create a new public room'), type: "createPublicRoom"});
+						results.push({ id: "create-public-room", displayName: t('spreed', 'New shared call'), type: "createPublicRoom"});
```

Comment:
I think most people will not understand what this means, click on this and then add people to their call, not knowing, that the call is publicly available if people guess the url....
Sharing is unclear in terms of being public or "i want to add users to this".